### PR TITLE
Docs：add Chinese and English documentation on how to join the fury community.

### PR DESCRIPTION
**Pull request by LofiSu**
Docs：add Chinese and English documentation on how to join the fury community.

### CODE REVIEW

**Comment by chaokunyang** [comment]
Could we don't copy all english documents? We could add tem when build websites, or forward to  english pages when chinese pages don't exist

**Comment by LofiSu** [comment]
> Could we don't copy all english documents? We could add tem when build websites, or forward to english pages when chinese pages don't exist

I ’m not sure what you're talking about.Do you mean no English documents?

**Comment by chaokunyang** [comment]
> > Could we don't copy all english documents? We could add tem when build websites, or forward to english pages when chinese pages don't exist
> 
> I ’m not sure what you're talking about.Do you mean no English documents?

Why we need to commit same file content duplicately to repo? We should avoid such things
image

**Comment by LofiSu** [comment]
> > > Could we don't copy all english documents? We could add tem when build websites, or forward to english pages when chinese pages don't exist
> > 
> > 
> > I ’m not sure what you're talking about.Do you mean no English documents?
> 
> Why we need to commit same file content duplicately to repo? We should avoid such things image


One is the English configuration and the other is the Chinese configuration, and the Chinese configuration has not been translated yet. I see that some of the other projects have default English, but there are also duplicate configurations in the en folder, but I think it can be deleted without much impact.

**Comment by LofiSu** [comment]
> > > Could we don't copy all english documents? We could add tem when build websites, or forward to english pages when chinese pages don't exist
> > 
> > 
> > I ’m not sure what you're talking about.Do you mean no English documents?
> 
> Why we need to commit same file content duplicately to repo? We should avoid such things image

The reason I copied and pasted the file here is that the Chinese translation can be translated directly on it in the future without creating a new file, and when converting to Chinese mode, no translated articles can be displayed in English.

**Comment by chaokunyang** [comment]
I don't think so. We will update English document frequently. It's very easy to introduce inconsistency. We should redirect to English document if document for other language doesn't exist

**Comment by LofiSu** [comment]
> I don't think so. We will update English document frequently. It's very easy to introduce inconsistency. We should redirect to English document if document for other language doesn't exist

ok I just delete them.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_join_community.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+```
+git remote add upstream https://github.com/apache/fury.git
+```
+
+### Step3：创建分支
+
+- 好了，现在可以开始贡献我们的代码了。fury 默认分支为 main  分支。无论是功能开发、bug 修复、文档编写，都请新建立一个分支，再合并到 main 分支上。使用以下代码创建分支：
+
+```
```
Comment (chaokunyang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_join_community.md:51]: 这里是否需要增加一个fork repo的步骤

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_join_community.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+- 填写这是什么类型的修改
+- 填写关联的 issue
+- 若有复杂变更，请说明背景和解决方案
+
+相关信息填写完成后，点击 Create pull request 提交。
+
+## **轻松步入 Fury 开源贡献之旅**
+
+"**good first issue**" 是一个在开源社区常见的标签，这个标签的目的是帮助新贡献者找到适合入门的问题。
```
Comment (LiangliangSui) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_join_community.md:96]: good first issue 感觉可以加一个link

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_join_community.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+如果你当前**有时间和意愿**参与到社区贡献，可以在 issue 里看一看 **good first issue**，选择一个感兴趣、适合自己的认领。
+
+## 拥抱 Apache Fury 社区
+
+在你为 Fury 贡献代码之余，我们鼓励你参与其他让社区更加繁荣的事情，比如：
+
+- 为项目的发展、功能规划 等提建议。
+- 创作文章、视频，开办讲座来宣传 Fury。
+- 撰写推广计划，同团队一同执行。
```
Comment (LiangliangSui) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_join_community.md:108]: 是否可以将我们的mail list和微信群二维码贴上

**Comment by RaymondLam1** [comment]
I noticed that fury's i18n documentation is very lacking. Maybe we should consider introducing LLM to translate related documents automatically.

**Comment by chaokunyang** [comment]
> I noticed that fury's i18n documentation is very lacking. Maybe we should consider introducing LLM to translate related documents automatically.

You mean deploy a llm in fury website? This needs gpu and is not affordable.
If the developers use llm to translate the doc, It don't involve changes in fury setup

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_join_community.md (hunk 3 above)
Comment (chaokunyang) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_join_community.md:108]: 同意，可以贴一下

**Comment by RaymondLam1** [comment]
> > I noticed that fury's i18n documentation is very lacking. Maybe we should consider introducing LLM to translate related documents automatically.
> 
> You mean deploy a llm in fury website? This needs gpu and is not affordable. If the developers use llm to translate the doc, It don't involve changes in fury setup

So we can only translate it by hand, and fortunately there are not many documents now. : )

**Review by chaokunyang — APPROVED** [review]

**Pull request merged**